go/interpreter/lfvm/memory.go

```diff
@@ -223,3 +134,30 @@ func (m *Memory) readWord(offset uint64, target *uint256.Int, c *context) error
 	target.SetBytes32(data)
 	return nil
 }
+
+// set copies the given value into memory at the given offset.
+// Expands memory as needed and charges for it.
+// Returns error if insufficient gas or offset+size overflows.
+func (m *Memory) set(offset, size uint64, value []byte, c *context) error {
+	data, err := m.getSlice(offset, size, c)
+	if err != nil {
+		return err
+	}
+	copy(data, value)
```

Comment:
How is this function supposed to behave if `value` is shorter than `size`? If zeros should be padded, this needs to be implemented.

Reply:
`getSlice` takes care of that.